README.md

```diff
@@ -127,9 +123,11 @@ npm install && npm run dev
 yarn install && yarn dev
 ```
 
-Your blog should be up and running on [http://localhost:3000](http://localhost:3000)! If it doesn't work, post on [GitHub discussions](https://github.com/vercel/next.js/discussions).
+Your blog should be up and running on [http://localhost:3000](http://localhost:3000)! You'll be able to find the studio in [http://localhost:3000/studio](http://localhost:3000/studio).
+
+If it doesn't work, post on [GitHub discussions](https://github.com/vercel/next.js/discussions).
```

Comment:
"If it doesn't work, post on…" sounds a bit too direct. I would suggest something more like "In case of any issues, you can…"